projects/storyteller/webapp/src/app/components/feed/feed.component.ts

```diff
@@ -0,0 +1,45 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+import { StoryList, Story,  Screenshot} from '@/core/proto';
+import { EncodingService, FirebaseService } from '@/core/services';
+
+import { from } from 'rxjs';
+@Component({
+  selector: 'app-feed',
+  templateUrl: './feed.component.html',
+  styleUrls: ['./feed.component.scss']
+})
+export class FeedComponent implements OnInit {
+  storylist: StoryList[];
+  screenshots: Screenshot[];
+
+  constructor(
+    private firebaseService: FirebaseService,
+    private encodingService: EncodingService,
+  ) {
+    if (firebaseService.isOnline) {
+      this.loadStory();
+      this.loadscreenshots();
+    } else {
+      this.firebaseService.anonymousLogin().then(() => {
+        this.loadStory();
+        this.loadscreenshots();
+      });
+    }
+  }
+  loadStory(): void {
+    this.firebaseService.getstorylistAll().subscribe(storylist => {
+      this.storylist = storylist;
+
+    });
+  }
+  loadscreenshots(): void {
+    this.firebaseService.getscreenshotAll().subscribe(screenshots => {
+      this.screenshots = screenshots;
+    });
+  }
+  ngOnInit() {
+  }
+
+}
+
+
```

Comment:
General comment, related to whole PR.
It has lack of code formatting.
First thing what you can do is to read angular style guide.
https://angular.io/guide/styleguide
It's a lot of text, but it's worth it.

Second thing. Your code editor actually can format your code automatically.
Try to use it and trust me, you're gonna love it.

Also we're going to create single base for all hasadna project. And tslint rules are going to be shared too.
If you don't know what is tslint it's ok. All you need to know is some of format issues, which are present in the PR will be fixed by new tslint rules.
btw tslint already work. You can run it:
```
ng lint
```
There're several errors.
To fix errors:
```
ng lint --fix
```

Reply:
But since it's still coming, could you fix the two issues manually?
* No empty line between methods
* More than one empty line
* 4 space indent style (should be 2)
* Import line spacing https://angular.io/guide/styleguide#import-line-spacing